scripts/fork/actions/beaconDrawLockAndNetworkTotalSupplyPush.ts

```diff
@@ -0,0 +1,112 @@
+import hre from "hardhat"
+import { Contract, PopulatedTransaction } from "ethers";
+import { ProviderOptions, ContractsBlob } from '../types'
+import { getContract } from "../helpers/getContract";
+import getJsonRpcProvider from "../helpers/getJsonRpcProvider";
+import { calculateBeaconDrawToPushToTimelock, calculateDrawTimestamps } from "../helpers";
+import getMultiTicketAverageTotalSuppliesBetween from "../helpers/get/getMultiTicketAverageTotalSuppliesBetween";
+import { sumBigNumbers } from "../helpers/sumBigNumbers";
+const debug = require('debug')('pt-autotask-lib');
+
+export interface PrizePoolNetworkConfig {
+    beaconChain: ProviderOptions;
+    allPrizePoolNetworkChains: ProviderOptions[];
+}
+  
+  export async function beaconDrawLockAndNetworkTotalSupplyPush(
+    contracts: ContractsBlob,
+    config: PrizePoolNetworkConfig
+  ): Promise<PopulatedTransaction | undefined> {
```

Comment:
I don't think we should be able to return `undefined`. It should at least return an error.